build.gradle

```diff
@@ -77,10 +137,14 @@ repositories {
     maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/"}
 }
 
+configurations.all {
+    resolutionStrategy.force("jakarta.json:jakarta.json-api:2.1.1")
+}
+
 dependencies {
 
     def opensearchVersion = "3.0.0-SNAPSHOT"
-    def log4jVersion = "2.20.0"
+    def log4jVersion = "2.17.1"
```

Comment:
Just for my understanding why log4j has been degraded?

Reply:
This is to match the version in OpenSearch

Reply:
Looks like it's failing

Reply:
I could potentially try a resolutionStrategy too if we want to go that route

Reply:
Are we facing any issue with `2.20.0`? SDK should be independent of OpenSearch.